src/components/DocumentationTopic.vue

```diff
@@ -106,6 +107,7 @@ import BetaLegalText from 'theme/components/DocumentationTopic/BetaLegalText.vue
 import LanguageSwitcher from 'theme/components/DocumentationTopic/Summary/LanguageSwitcher.vue';
 import DocumentationHero from 'docc-render/components/DocumentationTopic/DocumentationHero.vue';
 import WordBreak from 'docc-render/components/WordBreak.vue';
+import { TopicStyles } from 'docc-render/constants/TopicStyles';
```

Comment:
Should we change this to `TopicSectionsStyle` to match the JSON key and better reflect what it means?

Reply:
That constant would be used for the `@Links` directive as well, as those use the same spec. But I guess it makes more sense to rename it? LMK what you think, I am fine either way.

Reply:
I just think `TopicStyles` is a little bit vague on its own. In the JSON it's called `topicSectionsStyle` and it's called `@TopicsVisualStyle` in the markdown and both of those are slightly more descriptive of what this string represents. Not a blocker.

Reply:
Done.